src/main/java/org/sonarlint/intellij/analysis/DefaultSonarQubeRunnerFacade.java

```diff
@@ -17,42 +17,46 @@
  * License along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
  */
-package org.sonarlint.intellij.inspection;
+package org.sonarlint.intellij.analysis;
 
 import com.intellij.openapi.application.ApplicationInfo;
 import com.intellij.openapi.components.AbstractProjectComponent;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.project.ProjectCoreUtil;
+
 import java.io.File;
 import java.util.Properties;
 import javax.annotation.CheckForNull;
+
 import org.jetbrains.annotations.NotNull;
 import org.sonar.runner.api.EmbeddedRunner;
 import org.sonar.runner.api.IssueListener;
 import org.sonar.runner.api.LogOutput;
 import org.sonarlint.intellij.config.SonarLintGlobalSettings;
 import org.sonarlint.intellij.config.SonarLintProjectSettings;
-import org.sonarlint.intellij.console.SonarLintConsole;
+import org.sonarlint.intellij.ui.SonarLintConsole;
 import org.sonarlint.intellij.util.SonarLintConstants;
+import org.sonarlint.intellij.util.SonarLogOutput;
 
-public final class SonarQubeRunnerFacade extends AbstractProjectComponent {
+public final class DefaultSonarQubeRunnerFacade extends AbstractProjectComponent implements SonarQubeRunnerFacade {
 
   private final SonarLintProjectSettings projectSettings;
   private final SonarLintConsole console;
   private boolean started;
   private EmbeddedRunner runner;
 
-  protected SonarQubeRunnerFacade(Project project, SonarLintProjectSettings projectSettings, SonarLintConsole console) {
+  protected DefaultSonarQubeRunnerFacade(Project project, SonarLintProjectSettings projectSettings, SonarLintConsole console) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Protected method '<init>(...)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUnusedProtectedMethod)